### PR TITLE
docs: add a clarification about disabling password logins

### PR DESCRIPTION
**Pull request by alexyarosh**
using phrasing suggested in [this thread](https://metaboat.slack.com/archives/C013N8XL286/p1728423886248479)

### CODE REVIEW

File: docs/people-and-groups/authenticating-with-saml.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ Metabase accounts created with an external identity provider login don't have pa
 
 ## Disabling password logins
 
-> **Avoid locking yourself out of your Metabase!** Turning off password logins applies to all Metabase accounts, _including your Metabase admin account_. We recommend that you keep password authentication **enabled**.
+> **Avoid locking yourself out of your Metabase!** Turning off password logins applies to all Metabase accounts, _including your Metabase admin account_. Make sure you can log in your admin account using SSO before disabling password logins.
```
Comment (jeff-bruemmer) [inline docs/people-and-groups/authenticating-with-saml.md:184]:
```suggestion
> **Avoid locking yourself out of your Metabase!** Turning off password logins applies to all Metabase accounts, _including your Metabase admin account_. Before turning off password logins, make sure you can log in to your admin account using SSO.
```

**Review by jeff-bruemmer — APPROVED** [review]
Thanks @alexyarosh!

**Comment by alexyarosh** [comment]
@metabase-bot backport release-x.50.x

**Comment by alexyarosh** [comment]
@metabase-bot backport release-x.50.x

**Pull request merged**